src/Types.hs

```diff
@@ -319,12 +318,16 @@ newline :: RichText
 newline = "\n"
 
 showRecipe :: (String -> String) -> Recipe -> Maybe Int -> RichText
-showRecipe t r maybeServings =
```

Comment:
Why did this code change? Was it intentional? Does it have to do with the rest of this PR?

Reply:
Ah, I see. Can we keep the formatting closer to what it previously was so that the change is clearer from the diff?

Reply:
Sure thing, will fix now.